packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -159,6 +159,7 @@ export interface SummaryTreeBuilderParams {
 	groupId?: string;
 }
 /**
+ * SummaryTreeBuilder is a helper class following the builder pattern that helps build a summary tree.
```

Comment:
Nit: I've been recommending not starting descriptions with the name of the item being documented, since it's a bit redundant. I would recommend:
```suggestion
 * A helper class that helps build a summary tree.
 * @remarks Uses the builder pattern.
```